airflow/providers/google/cloud/hooks/cloud_memorystore.py

```diff
@@ -861,7 +861,7 @@ def update_parameters(
         :param update_mask: Required. Mask of fields to update.
             If a dict is provided, it must be of the same form as the protobuf message
             :class:`~google.protobuf.field_mask_pb2.FieldMask`
-            Union[Dict, google.protobuf.field_mask_pb2.FieldMask]
+            Union[dict, google.protobuf.field_mask_pb2.FieldMask]
```

Comment:
```suggestion
```

Not needed since we have type annotations in the function.

Reply:
ty